### ISSUE TITLE
Timezone for expiration dates/times of sharing links

### ISSUE THREAD

**Issue by ghost** [owncloud/core#8363]
In /lib/base.php the default time zone is set to UTC. I can change the log timezone by adding a line in the config/config.php. And in the calendar I can set a timezone for events etc.

But none of these settings has any effect on the timezone for expiration dates/times of (public) sharing links. They're still UTC.

Is it possible to have owncloud _set the global/overall default timezone to the timezone of the server_ it's running on? Or give (admin) users the option to _set the global/overall timezone in the config.php_? I think this would resolve a lot of timezone issues.

It's not a big thing but it would be cool to see sharing links expire according to my timezone, not UTC.

**Comment by nickvergessen** [comment]
Well you can set that by setting the (php) servers timezone.

**Comment by tomneedham** [comment]
So essentially, when you create the expiration time, you are setting it relative to your timezone, which needs to be converted to server timezone before being saved.

**Comment by dsthode** [comment]
Not quite, the expiration time I entered is relative to my timezone and the server's (in this case is the same one, "Europe/Madrid"), but as ownCloud converts the expiration time to UTC, it does not expire when intended, but two hours later (as "Europe/Madrid" is currently UTC+2)
